plugins/modules/win_file_version.ps1

```diff
@@ -19,7 +19,7 @@ If (-Not (Test-Path -LiteralPath $path -PathType Leaf)) {
 }
 $ext = [System.IO.Path]::GetExtension($path)
 If ( $ext -notin '.exe', '.dll') {
-    Fail-Json $result "Specified path $path is not a valid file type; must be DLL or EXE."
+    Fail-Json $result "Specified path $path is not a valid file type; Must be DLL or EXE."
```

Comment:
While I doubt someone is checking this string it is best to keep it as before. This is so someone doing the below will continue to work

```yaml
- community.windows.win_file_version:
    path: ...
  register: file_info
  failed_when:
  - file_info is failed
  - not file_info.msg is search('.* must be DLL or EXE.*')
```

```suggestion
    Fail-Json $result "Specified path $path is not a valid file type; must be DLL or EXE."
```